src/components/Main/Main.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useContext } from 'react';
+import { TodoList } from '../TodoList/TodoList';
+import { DispatchContext, TodosContext } from '../../Store';
+import { Todo } from '../../Types/Todo';
+
+export const Main: React.FC = () => {
+  const {
+    todos,
+    todosCompleted,
+    selectedActive,
+    selectedCompleted,
+  } = useContext(TodosContext);
+  const dispatch = useContext(DispatchContext);
+
+  const handleChange = () => {
+    dispatch({ type: 'todosCompleted' });
+
+    if (!todosCompleted) {
+      dispatch({ type: 'changeTodosCompletedTrue' });
+    }
+
+    if (todosCompleted) {
+      dispatch({ type: 'changeTodosCompletedFalse' });
+    }
+  };
+
+  const filterTodos = (items: Todo[]) => {
+    if (selectedActive) {
```

Comment:
The 'selectedActive' context variable is not clear. Consider renaming it to 'filterActive' or 'showActiveTodos' for better readability.